### PR TITLE
Add Mobile campaign url builders

### PR DESCRIPTION
**Pull request by diminishedprime**
Changes are staged here:

https://ga-dev-tools-integration.web.app/campaign-url-builder/
https://ga-dev-tools-integration.web.app/campaign-url-builder/play/
https://ga-dev-tools-integration.web.app/campaign-url-builder/ios/

https://ga-dev-tools-integration.web.app/ga4/campaign-url-builder/
https://ga-dev-tools-integration.web.app/ga4/campaign-url-builder/play/
https://ga-dev-tools-integration.web.app/ga4/campaign-url-builder/ios/

### CODE REVIEW

File: src/components/CampaignURLBuilder/Web/index.tsx [hunk 1]
```diff
@@ -316,7 +313,6 @@ export const CampaignUrlBuilder: React.FC<{ version: GAVersion }> = ({
           </TableRow>
         </TableBody>
       </Table>
-
       <Typography variant="h2">Related Resources</Typography>
       <Typography variant="body1" component="ul">
         <li>
```
Comment (pfrisella) [inline src/components/CampaignURLBuilder/Web/index.tsx:318]: Can you fix the links. Some are duplicates here.

Also, no related resource links for Android, iOS?

**Review by pfrisella — APPROVED** [review]

File: src/components/CampaignURLBuilder/Web/index.tsx (hunk 1 above)
Comment (diminishedprime) [reply src/components/CampaignURLBuilder/Web/index.tsx:318]: Removed the duplicate link. 

I was going to wait on related resources for once I had a better sense of where these are linked to/from, etc.

Comment (pfrisella) [reply src/components/CampaignURLBuilder/Web/index.tsx:318]: Update the links to remove the visit_id query parameter. There are 2 links in constants that include that param and shouldn't.

**Review by pfrisella — CHANGES_REQUESTED** [review]

**Review by pfrisella — APPROVED** [review]
LGTM

**Pull request merged**